.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ""
+labels: ""
+assignees: ""
```

Comment:
Should the md file call out "required" vs "optional" values?

Reply:
what's the intent for a bug filer to set 'assignees'. Is it to get the assignees' attention?